jupyter_server/serverapp.py

```diff
@@ -1469,7 +1469,8 @@ def initialize(self, argv=None):
         self.init_webapp()
         self.init_terminals()
         self.init_signal()
-        self.init_server_extensions()
+        if load_extensions:
+            self.init_server_extensions()
```

Comment:
I think it might be a good idea to log an informational message when `load_extensions` is False stating that "<extension_name> is running in exclusive mode" (or whatever terminology is appropriate).  This way any confusion as to why other extensions aren't appearing is more clear.

Likewise, I think a debug message in [`init_server_extensions()`](https://github.com/jupyter/jupyter_server/pull/76/files#diff-8a07fc4bce4a1b879d161c56f1524b2cL1417) stating which extension is being loaded would also be helpful information.  This way admins can confirm their configurations are behaving as expected.

Reply:
Added some logging messages.